config/prod.env.js

```diff
@@ -1,4 +1,5 @@
 'use strict'
 module.exports = {
-  NODE_ENV: '"production"'
+  NODE_ENV: '"production"',
+  API_URL: '"https://good-bread-api-staging.herokuapp.com/api/v1"'
```

Comment:
This file stores env variables needed. It's not git ignored so any secret variables will be kept on the back end